deploy/clusterrole.yaml

```diff
@@ -85,17 +82,32 @@ rules:
   - get
   - create
 - apiGroups:
-  - apps
-  resourceNames:
-  - istio-operator
+  - policy
   resources:
-  - deployments/finalizers
+  - poddisruptionbudgets
   verbs:
-  - update
+  - '*'
 - apiGroups:
-  - install.istio.io
+  - rbac.authorization.k8s.io
   resources:
+  - clusterrolebindings
+  - clusterroles
+  - roles
+  - rolebindings
+  verbs:
   - '*'
-  - istiocontrolplanes
+- apiGroups:
+  - ""
+  resources:
+  - configmaps  
+  - endpoints
+  - events
+  - namespaces
+  - jobs
```

Comment:
Is the operator installing any jobs?  I suspect this should be removed.  If it is, we should move the work being done by the job into the operator itself.

Reply:
Good point. I saw it in the log to query `jobs.batch is forbidden`

My env is a bit tossed, let me test it a bit more.

Reply:
not seeing jobs after installing the default cr.  going to remove it for now.